tests/tests.py

```diff
@@ -148,16 +183,16 @@ def test_custom_s3_policy(self):
                     's3:Put*',
                     's3:List*',
                     's3:Delete*'],
-                'Resource': 'arn:aws:s3:::moj-test-dev-static/*',
-                            'Effect': 'Allow',
-                            'Principal': {'AWS': '*'}
+                'Resource': resource_value,
+                'Effect': 'Allow',
+                'Principal': {'AWS': '*'}
             }
         ]
 
         project_config = ProjectConfig('tests/sample-project.yaml', 'dev')
 
         project_config.config['s3'] = {
-            'static-bucket-name': 'moj-test-dev-static',
+            'StaticBucketName': 'moj-test-dev-static',
```

Comment:
I don't think this should have changed - this is the input yaml format which should remain as it was.